app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+    file_part = str(command_list[1])
+    file_out_part = str(command_list[2])
```

Comment:
The extraction of `file_part` and `file_out_part` assumes that the command string is always correctly formatted. Consider adding error handling to manage cases where the command string does not contain the expected number of parts.